packages/shared/forks/ReactFeatureFlags.test-renderer.native.js

```diff
@@ -67,11 +67,12 @@ export const enableTransitionTracing = false;
 
 export const useModernStrictMode = false;
 export const enableDO_NOT_USE_disableStrictPassiveEffect = false;
+export const enableFizzExternalRuntime = true;
 export const enableDeferRootSchedulingToMicrotask = false;
 
 export const enableAsyncActions = true;
 
-export const alwaysThrottleRetries = true;
```

Comment:
fwiw, i think if this is true and working we should keep it true to prevent any regressions that would sneak in before we can land the flag in the main fbsource bundle.

Reply:
I would feel better about that if I had confidence in the timeline with which we could enable this internally. However, I don't have that, and I would be worried about the possibility that tests are written against the current (non-final) behavior.

Reply:
In other words... testing something that we'll never ship is worse to me than testing something that is currently in production (even if it is something we want to move away from eventually).